fabric-common/test/Commit.js

```diff
@@ -106,6 +106,18 @@ describe('Commit', () => {
 	});
 
 	describe('#send', () => {
+		it('throws if targets committers is missing', async () => {
+			const stub = sinon.stub(channel, 'getTargetCommitters').callsFake(() => {
+				return undefined;
+			});
+			commit.build(idx);
+			const request = {
+				targets: [fakeCommitter]
+			};
```

Comment:
Looking at Channel.getTargetCommitters(), this will never return `undefined`. I think the error condition we should be testing for is where the request targets is an empty array, which means an empty array will be returned from Channel.getTargetCommitters(), and won't require that method to be stubbed.